website/src/docs/hotchocolate/security/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Overview"
+---
+
+In this section we will learn how to secure our GraphQL endpoint.
+
+# Authentication
+
+Authentication in Hot Chocolate is built around the official authentication mechanisms in ASP.NET Core, allowing us to fully embrace their customizability and variety of authentication providers.
+
+[Learn more about authentication](/docs/hotchocolate/security/authentication)
+
+# Authorization
+
+Authorization is one of the most basic security concepts. It builds on top of authentication and allows us to restrict access to types and fields, based on whether a user is authenticated, assigned specific roles or satisfies one or more policies. Hot Chocolate closely matches and nicely integrates with the official ASP.NET Core authorization APIs.
+
+[Learn more about authorization](/docs/hotchocolate/security/authorization)
+
+<!-- # Persisted Queries
```

Comment:
ping

Reply:
I commented it out on Arif's request, since the docs of it aren't ready yet. Will comment it in again in the persisted queries PR, once this one is merged.